pandas/core/strings.py

```diff
@@ -2074,8 +2074,9 @@ def cat(self, others=None, sep=None, na_rep=None, join=None):
 
             If others is None, the method returns the concatenation of all
             strings in the calling Series/Index.
-        sep : string or None, default None
-            If None, concatenates without any separator.
+        sep : string, default ''
```

Comment:
string --> str; iirc when specifying parameter types we want to use the Python function name (e.g. bool instead of boolean, dict instead of dictionary, etc.).  Can you also modify na_rep a couple lines below?